config/helm/chart/default/templates/Common/operator/install-config.yaml

```
@@ -8,6 +8,7 @@ metadata:
 data:
   modules.json: |
     {
+      "csidriver": {{ .Values.csidriver.enabled }},
```

```suggestion
      "csiDriver": {{ .Values.csidriver.enabled }},
```
just curious, why `activeGate` and `oneAgent` :)